licenses.go

```diff
@@ -67,7 +67,7 @@ func checkMenderCompliance() error {
 		return err
 	}
 
-	args := []string{path.Join(pathToTool, "check_license_go_code.sh")}
+	args := []string{path.Join(pathToTool, "check_license_source_files.sh")}
```

Comment:
Should the old file be removed now?

Reply:
Yes, and it is, no?

Reply:
Lol, yes, it is! Github fooled me, I was looking for the "negative diff", but of course it is hidden behind "Load diff".